loom/common/platform/platformMobileIOS.mm

```diff
@@ -337,35 +369,64 @@ bool platform_wasOpenedViaRemoteNotification()
 ///checks if a given sensor is supported on this hardware
 bool platform_isSensorSupported(int sensor)
 {
-    ///TODO: 1844: Support sensors on iOS
-    return false;
+    BOOL supported = false;
+    
+    switch (sensor)
+    {
+        case 0:
+            supported = [mobileiOS isAccelerometerAvailable];
+            break;
+    }
+    
+    return supported;
 }
 
 ///checks if a given sensor is currently enabled
 bool platform_isSensorEnabled(int sensor)
 {
-    ///TODO: 1844: Support sensors on iOS
-    return false;
+    BOOL enabled = false;
+    
+    switch (sensor)
+    {
+        case 0:
+            enabled = [mobileiOS isAccelerometerAvailable];
```

Comment:
Yeah, it's definitely a typo on my part. Should have had `enabled = [mobileiOS isAccelerometerActive];`, will be fixed.